src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -92,13 +92,27 @@
            class="dropdown-item"
            routerLink="/hosts">Hosts</a>
       </li>
+      <li routerLinkActive="active"
+          class="tc_submenuitem tc_submenuitem_cluster_inventory"
+          *ngIf="permissions.hosts.read">
```

Comment:
Not sure if we should hide it if the orchestrator is not available.
For example for the Prometheus configuration we hide the menus.

Reply:
These two menu items are hidden when the user has no `HOSTS.read` permission. If orchestrator is unavailable, the information about how to enable it is displayed.